src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -98,7 +98,7 @@ export class EditDataGridPanel extends GridParentComponent {
 		@ILogService protected logService: ILogService
 	) {
 		super(contextMenuService, keybindingService, contextKeyService, configurationService, clipboardService, queryEditorService, logService);
-		this.nativeElement = document.createElement('editdatagridpanel');
+		this.nativeElement = document.createElement('editDataGridPanel');
```

Comment:
Again, in general it's best practice not to create custom named elements like this. Just make this a div and give it an ID of editDataGridPanel if you need to identify it.

Reply:
I tried to do that (create element as div), but the issue was that the table did not load at all when I did it that way for some reason.

Reply:
I don't understand what that means - styles shouldn't be changing when you attach something. Can you push a commit that shows what you tried?